ivy/functional/frontends/torch/pointwise_ops.py

```diff
@@ -450,3 +450,8 @@ def hypot(input, other, *, out=None):
 @to_ivy_arrays_and_back
 def sigmoid(input, *, out=None):
     return ivy.sigmoid(input, out=out)
+
+
+@to_ivy_arrays_and_back
+def lerp(start, end, weight):
+    return ivy.lerp(start, end, weight)
```

Comment:
ivy.lerp doesn't exist in the ivy API, you can suggest its addition by following the steps here: https://lets-unify.ai/ivy/deep_dive/ivy_frontends.html#missing-ivy-functions.
That being said the function can simply composed by a few arithmetic operations as described here: https://pytorch.org/docs/stable/generated/torch.lerp.html.
Also please add the `out` argument